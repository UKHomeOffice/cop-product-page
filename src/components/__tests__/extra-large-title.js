import React from "react";
import renderer from "react-test-renderer";
import ExtraLargeTitle from "../extra-large-title";

describe("extra-large-title", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<ExtraLargeTitle/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("Returns the correct title 'text' prop passed in", () => {
    const tree = renderer
      .create(<ExtraLargeTitle text={"a title"}/>);
    const testInstance = tree.root;
    expect(testInstance.props.text).toBe("a title");
  });
});