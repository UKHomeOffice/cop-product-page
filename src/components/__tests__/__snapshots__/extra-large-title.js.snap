// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`extra-large-title renders correctly 1`] = `
<div
  className="govuk-heading-xl"
/>
`;
